### PR TITLE
docs(readme): new logo and header

### PR DESCRIPTION
**Pull request by lidel**
Updates header and logo to match https://github.com/ipfs/boxo/pull/405

### CODE REVIEW

**Review by hacdias — APPROVED** [review]
🎉

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
-
-![kubo, an IPFS node in Go](https://ipfs.io/ipfs/bafykbzacecaesuqmivkauix25v6i6xxxsvsrtxknhgb5zak3xxsg2nb4dhs2u/ipfs.go.png)
+<h1 align="center">
+<img src="https://user-images.githubusercontent.com/157609/250148884-d6d12db8-fdcf-4be3-8546-2550b69845d8.png" alt="Kubo logo" title="Kubo logo" width="200">
```
Comment (Jorropo) [inline README.md:2]:
```suggestion
<img src="https://ipfs.io/ipfs/QmTtCH31LdGV4LjRm5FNmnw6hUwZq1Z1YXtjeNKHBeaRcQ/kubo-logo.png" alt="Kubo logo" title="Kubo logo" width="200">
```
```
curl https://user-images.githubusercontent.com/157609/250148884-d6d12db8-fdcf-4be3-8546-2550b69845d8.png | ipfs add --raw-leaves --stdin-name kubo-logo.png -w
```

**Review by Jorropo — CHANGES_REQUESTED** [review]
Disapointed, 0/20.

File: README.md (hunk 1 above)
Comment (Jorropo) [reply README.md:2]: Also, I don't really care but don't we have an SVG of the logo ?

**Review by BigLep — APPROVED** [review]

**Pull request merged**